### PR TITLE
Fix missing link color in WhatsNew panel

### PR DESCRIPTION
**Pull request by linhpha**
Fixed issue when `colorPalette.blue` returned empty list

### CODE REVIEW

File: platforms/intellij/compose/src/jvmMain/kotlin/foundry/intellij/compose/markdown/ui/MarkdownPanel.kt
[the remark this answers is not in the document]
Comment (linhpha) [reply platforms/intellij/compose/src/jvmMain/kotlin/foundry/intellij/compose/markdown/ui/MarkdownPanel.kt:1]: updated!

File: platforms/intellij/compose/src/jvmMain/kotlin/foundry/intellij/compose/markdown/ui/MarkdownPanel.kt [hunk 1]
```diff
@@ -148,9 +148,9 @@ private val JewelTheme.Companion.linkColor: Color
   @Composable
   get() {
     return if (isDark) {
-      colorPalette.blue.lastOrNull() ?: Color.Blue
+      colorPalette.blue.lastOrNull() ?: Color.White
     } else {
-      colorPalette.blue.firstOrNull() ?: Color.Blue
+      colorPalette.blue.firstOrNull() ?: Color.White
```
Comment (ZacSweers) [inline platforms/intellij/compose/src/jvmMain/kotlin/foundry/intellij/compose/markdown/ui/MarkdownPanel.kt:153]: hardcoding it to white will be bad on light themes :)

Comment (kateliu20) [reply platforms/intellij/compose/src/jvmMain/kotlin/foundry/intellij/compose/markdown/ui/MarkdownPanel.kt]: Makes sense 😅

**Review by kateliu20 — APPROVED** [review]

File: platforms/intellij/compose/src/jvmMain/kotlin/foundry/intellij/compose/markdown/ui/MarkdownPanel.kt (hunk 1 above)
Comment (linhpha) [reply platforms/intellij/compose/src/jvmMain/kotlin/foundry/intellij/compose/markdown/ui/MarkdownPanel.kt]: ahh that's tricky 😄 I switched to use JewelTheme text color instead

**Pull request merged**